<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossSettingsWidget</class>
 <widget class="QDialog" name="CrossSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_root">
       <property name="text">
        <string>SDKRoot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_root"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_root">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_gnu32">
       <property name="text">
        <string>Mipsel-linux-gnu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_gnu32"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_gnu32">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_gnu64">
       <property name="text">
        <string>Mips64el-linux-gnu</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_gnu64"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_gnu64">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_llvm">
       <property name="text">
        <string>llvm</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_llvm"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_llvm">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_qemu">
       <property name="text">
        <string>Qemu</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_qemu"/>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_qemu">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_gnulinux">
       <property name="text">
        <string>Gnu-linux</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_gnulinux"/>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_gnulinux">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_target">
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_target"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
